### ISSUE TITLE
Pushing new data within Vue 3 breaks Tabulator (Reactive Data)

### ISSUE THREAD

**Issue by megasteve19** [olifolkerd/tabulator#3822]
**Describe the bug**
Pushing new data breaks Tabulator. Any changes made inside a method causes infinite loop. You can look at codepen.

**Tabulator Info**
5.2

**Working Example**
https://codepen.io/megasteve19/pen/gOvVqXY

**To Reproduce**
A step by step guide to recreate the issue in your JS Fiddle or Codepen:
1. Open console
2. Click "Add row" button.
3. You will see 'Looping.' log from 'rowFormatter'.

**Expected behavior**
Updating table data.

**Screenshots**
 [N/A]

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Browser: Firefox

**Smartphone (please complete the following information):**
 [N/A]

**Additional context**
 [N/A]

**Comment by ejblom** [comment]
I experienced the same when I used data directly from the store in my tables. This wasn't an issue, but I noticed that adding new data didn't update the Tabulator code. Luckily, the reactivity option (http://tabulator.info/docs/5.2/reactivity) does provide for this behavior. But when I enable the option and add something to the array provided as the data source to Tabulator, the page completely hangs and crashes.
